requirements.txt

```diff
@@ -1 +1,2 @@
 librosa>=0.4.3
+tensorflow>=0.12
```

Comment:
Does pip complain if you try to install tensorflow when tensorflow-gpu is already installed? (tensorflow's pip doesn't work for me and I don't have a GPU, so I can't test it). If that's the case, leaving the CPU version would mean that people just downloading the repo and testing it won't have issues and won't complain about cuda errors, and I assume people who have a decent enough GPU already have tensorflow or know how to get it. If pip allows installing both of them and then everything breaks, it would probably be better to use the gpu version. But the CI doesn't have GPU, so we'd have to adapt the script.